docs/blog/2022-07-16-obs-feature.mdx

```diff
@@ -0,0 +1,199 @@
+Simplified data types should also make it easier to implement new file types (including via plugins).
+
+The former `raster` data type has been split into two new data types, `image` and `obsSegmentations`, to better reflect semantics (as opposed to exposing the implementation difference between bitmask and polygon segmentations). (Internally, the fact that an `obsSegmentations` file type may return either polygons or bitmasks is handled by returning an `obsSegmentationsType` from the data loader class.)
+
+For the full list of data types, please visit the updated [Data Types and File Types](/docs/data-types-file-types/) page or the new [File type implementation](/docs/dev-add-loader/) page.
+
+### Dataset definitions
+
+Dataset definitions ([`datasets`](/docs/view-config-json/#datasets) in the view configuration) are no longer limited to one file per data type.
+
+Each `dataset` may now contain multiple file definitions corresponding to the same data type.
+
+### Mapping between files and views
+
+File lookup from within a view now takes into account `(dataset, dataType, viewCoordinationValues)`.
+
+
+The `viewCoordinationValues` item refers to the coordination values that a particular view takes on at a given point in time.
+This is required to disambiguate between multiple files within a dataset which have the same `dataType` (now that each dataset may contain multiple files per data type).
+
+The diagram below shows how the matching between views and files would be performed for a configuration containing one dataset and three views.
+
+<MatchingDiagramTabs />
+
+### File definitions
+
+[File definitions](/docs/view-config-json/#files) no longer require the `type` (data type) property.
+Instead, we now store a [mapping](https://github.com/vitessce/vitessce/blob/main/src/app/constant-relationships.js) from `fileType`s to data types internally and do the lookup automatically.
+
+### Joint file types
+
+[Joint file types](/docs/data-types-file-types/#joint-file-types) can now be defined via functions which expand one file definition into an array of file definitions.
+
+
+### Exported constant values
+
+Exported [constants](/docs/constants/) have been updated to reflect the changes to file types, data types, and coordination types as described here.
+
+While no longer documented, we will continue to export the old constants for backwards compatibility (since their values will remain useful when  defining view configs corresponding to previous schema versions).
+However, instead of implementing them using plain JS objects, deprecated constants will be exposed via [proxies](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy) which log deprecation messages to the console when accessed.
+
+### VitessceConfig constructor
+
+The [VitessceConfig](/docs/view-config-js/) constructor now requires `schemaVersion` as its first parameter.
```

Comment:
> Is this a common idiom?

We already require the `version` property in the JSON schema, so I do not think requiring it here is much different.
I think it is similar to Vega/Vega-Lite's  [`$schema`](https://vega.github.io/vega/docs/specification/ ) property.

I suppose Vitessce could accept only the current schema but then, exactly, it would be painful to upgrade Vitessce if you have many configs. With `version` we can handle schema upgrades automatically because it indicates which version of the schema (and the exact Vitessce behavior) was assumed at the time the config was written.